shell/app/modules/project/router.tsx

```diff
@@ -75,11 +78,29 @@ function getProjectRouter(): RouteConfigItem[] {
             },
             {
               path: 'task',
-              tabs: COLLABORATE_TABS,
-              ignoreTabQuery: true,
-              getComp: (cb) => cb(import('project/pages/issue/task')),
-              layout: {
-                noWrapper: true,
+              render: (props) => {
+                const { location } = props;
+                const { search } = location;
+                const params = qs.parse(search);
+                return <Redirect to={`all?${qs.stringify({ ...params, type: 'TASK' })}`} />;
```

Comment:
Is this relative path ?

Reply:
Yes, it should be the relative path of this route.